src/components/ProductDetails/ProductDetails.component.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import './ProductDetails.component.scss';
+import endpoints from '../../api/endpoints.config';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'react-bootstrap';
```

Comment:
Some of these imports are never used. Consider using a `eslint` and `eslint-plugin-react`.

Reply:
Updated this, sorry I didnt realize it